# Chapter 13

RNA polymerase II is responsible for transcribing genes that code for proteins.

In eukaryotes, the term promoter is used to describe all the sequences that are important in initiating a transcription of a gene. There are several of these, some called **core promoter** but also **upstream promoter elements** which lie upstream of the core promoter.

**ALternative promoters** are used in different tissues and direct synthesis of mRNAs of different length. Can also be called **multiple promoters**.

### Synthesis of eukaryotic RNA is regulated primarily by activator proteins

Transcription initiation can be influenced by DNA-binding proteins that recognize specific sequences located near the attachment site for RNA polymerase. Eukaryotic genes have a low rate of transcription initiations, regardless of how strong the promoter is. In order to achieve effective initiation, formation of the complex must be activated by additional proteins. What complex? Polymerase 2 and 3!

Bacteria have only the RNA polymerase, not different varieties of it. Eukaryotes employ a different strategy for transcription initiation where activators are playing a much more prominent role than repressor proteins.

[Here is a list of RNA classifications](https://github.com/stigpaulsberg/Genome-4/blob/master/Content/Chapter_12/RNA-classes.PNG?raw=true)

#### Transcription factors to note:

- **General transcription factor (GTF)**:

This is a complex MADE UP OF THE TWO CLASSES BELOW

- **TATA-binding protein (TBP)**:

This has a saddlelike shape that wraps partially around the double helix of DNA, forming a platform onto which the RNA polymerase is subsequently positioned.

- **TBP-associated factors (TAFs)**:

These assist the TBP in binding to its sequence specific site.

**Preinitiation complex** is a big complex of multiple units including RNA polymerase. When this complex is set, with the help of transcription factors attached to enhancer sites and proximal binding sites, along with a mediator, the initiation complex goes through a phosphorylation that makes it possible for the RNA polymerase to leave the complex due to a change in its ionic properties. RNA polymerase can now transcribe the gene.

**Mediator** is a multisubunit protein that makes physical contact with the transcription factors, which in turn makes contacts with different components of the perinitiation complex.

[Here's a photo illustrating it all](https://github.com/stigpaulsberg/Genome-4/blob/master/Content/Chapter_12/Transcription%20and%20all%20that%20shit.PNG?raw=true)

#### Binding sites for transcription factors to note:

- **Proximal binding sites**:

The proximal binding sites are **upstream promoter elements**, most of which are located within 2 kb of the transcription start site of the target gene. Attachment of a transcription factor to one of these sites influences transcription only of the gene within whose promoter the element is located

- **Enhancers**:

This is also a *binding site*, but more distant. These can be positioned anywhere within the functional domain that contains their target gene. A *single enchanter* can influence **transcription of multiple genes within its domain**, but it is prevented by insulator sequences from affecting genes in neighboring domains.


**MOST TRANSCRIPTION FACTORS ARE ACTIVATORS, NOT REPRESSORS**, but there are some, called **silencers**.


## 12.3 DEGRADATION OF THE COMPONENTS OF THE TRANSCRIPTOME


#### Several processes are known for nonspecific RNA turnover

Half life of RNA is of interest.

Bacterial mRNA are turned over more rapidly than eukaryotic mRNA. Half life for EU-mRNA can be several hours for mammals, and half life for PRO-mRNA can be like 20 minutes or so.

Fewer studies have been done on noncoding RNA, but the studies that have been conducted suggest that tRNA and rRNA are both turned over more slowly than mRNA. **tRNA half-lives between 9 hours to several days** and **rRNA half-lives up to 8 days**.

In bacteria, nonspecific mRNA degradation is carried out by **degradosome**, a multiprotein structure whose components include polynucleotide phosphorylase (PNPase), which removes nucleotides sequentially from the 3'-end of an mRNA.

*The eukaryotic equivalent is called **exosome**, which comprises a *ring of six proteins*, each of which has ribonuclease activity, with three RNA-binding proteins attached to the top of the ring.*
Exosomes are present in both the cytoplasm and the nucleus of a eukaryotic cell. The main role is rapid turnover of aberrant RNAs that have not been processed or transcribed correctly. **Aberrant mRNAs are detected by a **surveillance mechanism**, which identifies ones that lack a termination codon, which might occur if the DNA has been copied incorrectly.
