# Overview - Chapter 3
## MAPPING Genomes

A genome map indicates positions and interesting features of genes and DNA sequences. There are two complementary approaches:

- **Genetic mapping** or **linkage analysis** is based on the use of genetic techniques like planned breeding experiments and examination of pedigrees (lineage, family tree).

- **Physical mapping** which uses biology techniques to examine DNA molecules directly to identify positions of sequence features including genes.

## Why is a genome map important?

The sequencing of DNA has a major limitation. Only with the most advanced methods are we able to sequence only 750 bp in a single experiment. This means that long sequences needs several experiments. One long equals many small. This is the **shotgun method**.

---

### Definitions

- **QTL** *(quantitative trait loci)* are regions of a genome. These regions can each possibly hold several genes that control variable traits that are favorable in agriculture, for example. This can be seedling number, meat productivity and pest resistance in crop plants.

---

### Summary

- **Genome maps** provide the framework for sequencing projects because they indicate the positions of genes and other recognizable features so it enables the accuracy of an assembled DNA sequence to be checked. It is something "*double check*" your, for example, DNA that you've sequenced in lab."

- Genome maps are used in methods for identifying the **functions** of genes involved in interesting traits, like diseases, and for identifying QTLS's that control traits such as meat productivity in farm animals.

- In the first genetic maps, the **markers** were genes whose alleles could be distinguished because they gave rise to easily recognized phenotypes such as different eye colors, shape of wings, or whose alleles could be distinguished by biochemical tests (like antibac resistance, or metabolism of different nutrients).

- Today, DNA markers are also extensively used for mapping these markers including restriction fragment length polymorphisms (*RFLPs*), simple sequence length polymorphisms (*SSLPs*), and single-nucleotide polymorphisms (*SNPs*), all of which can be typed quickly and easily by PCR. **PCR is a good method for DNA marking**

- The **relative positions** of genes and DNA markers on chromosomes are determined by *linkage analysis*, which enables the recombination frequency between a pair of markers to be calculated, providing the data needed to deduce the relative positions of the markers on the genetic map.

- For many organisms, linkage analysis is carried out by following the inheritance of markers in planned breeding experiments, but this is **not possible** for humans. Instead, genetic mapping of the human genome depends on examination of marker inheritance in large families, a procedure called **pedigree analysis**

- Genetic maps have relatively **poor resolution** and tend to be inaccurate and must be refined by *physical mapping* if the map is to be used in a genome sequencing project. So genetic maps are not enough to sequence a genome.

- The positions of restriction sites in a small DNA molecule can be determined by restriction mapping.

- Optical mapping enables the positions of restriction sites and some other sequence features to be directly visualized in a long DNA molecule. **This we have performed in our lab**.

- The most detailed physical maps are obtained by sequence-tagging site (**STS**) content mapping, which makes use of a mapping reagent, a collection of overlapping DNA fragments that span an entire chromosome or genome. The mapping reagent can be a *library of clones* (cDNA library?) or a radiation hybrid panel. 
