# Questions for the colloquium lecture 6 (chapter 8 and 10)



## How are eukaryotic organellar genomes organized? What makes them "special"?

*organellar **genomes** * : Most organellar genomes are circular. What makes them special is that they resemble the genomes of prokaryotes. Another thing is that there are many copies of the genome.



## What comprehends the endosymbiont theory and which proofs support it?

The endosymbiont theory is based on the idea that eukaryotic cells and the mitochondria were once two separate cells, but now live in unison.

There are two proofs that support it:

**1.** : Observations reveal that gene expression processes in organelles (mitochondria) are similar in many respect to those in bacteria

**2.** : Nucleotide sequences of organelles are more similar to those of bacteria than the eukaryotic equivalents.

There are also many different cells that contains energy producing organelles. They are highly autonomous but dependent on the nucleus of their eukaryotic partner for protein synthesis and such

## What are mitochondria and what is their function?

Mitochondria are thought of as prokaryotic cells. They live in a endosymbiont relationship with eukaryotes, and their function now is to produce energy in the form of ATP.

## How is mitochondrial DNA inherited?

Mitochondrial DNA is mostly inherited maternally, meaning this does not follow a Mendelian inheritance pattern.

Paternal mitochondria are degraded by enzymes.

## How are mitochondrial diseases different from nuclear inherited diseases?

The mother can be affected by a mitochondrial disease in a manner that only slightly expresses itself, or don't express itself at all. The meaning is that of all the mitochondria in the gamete of the mother might be divided between diseased mitochondria and healthy mitochondria. When these are passed on to the offspring, it is mere chance that dictates how affected the offspring will be.

Lets say 20% of the mitochondria the offspring inherits are diseased, then the child might be just fine. But if 80% of it is diseased, the child will be affected. But the ratio of healthy/diseased mitochondria that is passed on is merely a gamble. The number of diseased mitochondria varies from cell to cell.

## What are the structural and functional differences between euchromatin and heterochromatin?

Euchromatin | heterochromatin
- | -
DNA that has a relatively **open conformation**, either as the 30 nm fiber or simply as beads-on-a-string format.  | DNA that is still relatively **compact**. Less compact than metaphase though.
Comprises those parts of the chromosomal DNA that contain *active* genes --> RNA. | These are transcriptionally silent. --> no RNA :(. Containing few, if any, active genes.
RNA polymerase/proteins involved in transcription can access the material. | Can't access the material.
- | There are two types of heterochromatin.
- | **Constitutive**: A permanent feature of all cells. Represent DNA that contains no genes and so always retain compact organization. Most of human Y chromosome is made of constitutive heterochromatin.
- | **Facultative**: Not permanent feature but is seen in some cells some of the time. Contains genes that are inactive in some cells or at some periods of the cell cycle. When genes are *inactive*, compacted into heterochromatin.


## What does the territory structure of the nucleus comprehends?

The territory structure? Speckles, Cajal Bodies and fibrillarin.

When proteins a marked with fluorescence, distinct regions or compartments are clearly visible.

There are three regions that are evident:

**Nucleoli**: This can be visualized by labeling the fibrillarin protein with *GFP (green fluorescent protein)*. **Fibrillarin** is a component of small nucleolar ribonucleoporteins that are involved in rRNA processing.

**Speckles**: Labeling of the proteins involved in mRNA splicing shows that activity is localized in something called **Speckles**. These are widely distributed and less well defined than the nucleoli.

**Cajal bodies**: These are probably involved in synthesis of small nuclear and small nucleolar RNAs.

**The text above was probably not the answer to the question. The question might be revolved around chromosomes and their territories.**

Chromosomes seem to each have distinct territories they reside in in a nucleus. The chromosomes take up the majority of space in the nucleus but they are all separated from one another by **nonchromatin regions**. These are regions where enzymes and other proteins involved in *expression of the genome* are located.

There seems to be channels throughout the nucleus which are these **nonchromatin regions**.

The chromosome territories seem to be very static, but these territories *don't seem to be retained* in the nuclei of daughter cells. **BUT** there must be some association, some constraints on the territory locations. The reasoning behind it is the chromosome **translocations** (one segment of a chromosome attaches to another. This is more frequent between some pairs than others). For example, a translocation between human chromosome 9 and 22 is a common cause for chronic myeloid leukemia.

There is evidence that, at least in some organisms, certain chromosomes prefers to occupy territories close to the outline of the nucleus. *Relatively little genome expression occurs here*.

## Which chemical histone modifications exist?

The best studied of these are **histone acetylation**: the attachment of acetyl groups to *lysine amino acids* in the N-terminal regions of each of the core molecules. So the *tails* of the histone octamer is acetylated. Histones in heterochromatin are generally unacetylated whereas those in active domains are acetylated. This shows that *histone acetylation is closely linked to DNA packaging*. Enzyme: **Histone acetyltransferase (HATs).

**Histone deacetylation**: represses active regions of the genome. This is the role of **histone deacetylase (HDAC)**, which counteracts the **HAT**.

HDACs and HATs are contained in multiprotein complexes.


### Other types of histone (covalent) modifications

**Methylation** of lysine and arginine residues. This modification is thought to be irreversible and responsible for permanent changes to chromatin structures. This view has been challenged, but it is still accepted that the effects of methylation are relatively long-term.

**Methylation** happens on residues of lysine and arginine. Methylation was previously thought to be irreversible and hence a permanent modification of the chromatin structure. This is challenged, but it is still accepted that the effect of this are relatively long term. I think epigenetics and methylation of genes are linked together.

The purpose of methylation is to either repress or activate *gene expression*. The methylation of lysine-9 forms a binding site for the HP1 protein. This in turn induces DNA packaging and silences gene expression.

**Phosphorylation** of serine, threonine and tyrosine residues.

**Ubiquitination** of lysine residues at the C-terminal (acetylation is at the N-terminal) regions. This modification involves addition of the small, common (ubiquitous) protein called **ubiquitin**.

**Citrullination**. This is the conversion of arginine to the related amino acid called citrulline, by *replacement* of the terminal =NH group of the arginine *side chain* with and =O group.


## What is histone acetylation/ de-acetylation and which enzymes are involved?

**Histone acetylation/de-acetylation** is a mechanism done by big multiprotein complexes that is linked to DNA packaging and the activation/de-activation of genes.

**Acetylation**: the enzymes involved in the acetylation are **histone acetyltransferase (HATs)**. They are not alone in the acetylation but are part of a bigger multiprotein complex.

**De-acetylaton**: The enzymes involved in this are also part of a bigger multiprotein complex, but the main protein discussed are the **histone deacetylase (HDACs)**.

## What is nucleosome repositioning and how can it influences gene expression?

**Nucleosome repositioning** gives DNA-binding proteins access to attachment sites. **Remodeling** is not a covalent alteration to histone molecules. It is rather a energy-dependent process that weakens the contact between the nucleosome and the DNA associated with it. There are three types of changes that can occur:

**Remodeling**: This is a change in the general structure of the nucleosome, but not a repositioning. When the remodeling is done *in vitro* the nucleosome gains double the size and an increased DNase sensitivity.

**Sliding**: or ***cis***-**displacement** physically moved the nucleosome along the DNA.

**Trasnfer**: or ***trans***-**displacement** results in the nucleosome being transferred to a second DNA molecule.

These changes are involved in the sensitivity of the DNA to the DNase. And also making the attachment sites for DNA-binding proteins accessible.

## How is DNA methylated and what is the purpose of it?

**DNA Methylation** are changes that are associated with semi-permanent silencing of gene-regions. These methylations are carried out by **DNA methyltransferases**.

The methylation pattern **is not random**. Instead it is limited to the cytosine in some copies of the sequences 5'-CG-3' and, in plants, 5'-CNG-3'. There are *two types of methylation activity*.

The first is **maintenance methylation**. This happens after genome replication. It is responsible for adding methyl groups to the newly synthesized strand of DNA at positions opposite methylated of parent strand. This ensures that two daughter strands are identical to its parent.

The second is ** *de novo* methylation** which adds methyl groups at *totally new positions* and so changes the pattern of methylation in a localized region of the genome.

**Both of these methylations results in a repression of gene activity.**

## What is genomic imprinting?

Genomic imprinting is a form of **DNA methylation**. This is a feature in which only one pair of genes, present on homologous chromosomes in a diploid nucleus, is expressed while the second is silenced by *methylation*.

**X inactivation** is a form of imprinting to almost total inactivation of one of the X chromosomes in a female mammalian cell. If both the X chromosomes in females were expressed then proteins might be synthesized at twice the rate compared to males.

## What is a TAD?

**Topologically associated domain**. The architecture of these gives rise to many intradomain interaction but few interactions between domains.

TADs are present in all metazoan genomes. TADs can be looked at as *functional domains*: each TAD contains a set of genes that are subject to a similar expression pattern, which is set by the identities of regulatory sequences also present in the TAD.

When the genes it contains are active, a TAD adopts a relatively open conformation, presumably to aid access of proteins involved in transcription. When its genes are silent, the repressed TAD takes up a more compact organization.

----

# SUMMARY

- The nuclear environment has a substantial and important impact on the expression of the genome.

- A eukaryotic nucleus has a highly ordered internal architecture that includes structures associated with rRNA processing, mRNA splicing, and synthesis of small nuclear and small nucleolar RNAs.

- The most compact form of chromatin is *heterochromatin*, within which genes are *inaccessible* and cannot be expressed.

- The nucleus is thought to contain a fibrous matrix to which chromosomal DNA is attached, although the existence of this matrix has been questioned.

- Each chromosome has its own **territory** within the nucleus, these territories being separated from one another by *nonchromatin regions* within which the *enzymes and other proteins* involved in *genome expression* are located.

- A chromosome is made up of a series of *topological associated domains (TADs)*, each comprising a contiguous of chromatin *folded into coils and loops*.

- Each domain is delimited by a pair of *insulators*, which *maintain the independence of the domain*.

- Nucleosomes appear to be the **primary determinants** of *genome activity* in eukaryotes, not only by virtue of their positioning on a strand of DNA, but also because the *precise chemical structure* of the histone proteins contained within nucleosomes is the *major factor determining the degree of packaging* displayed by a segment of chromatin.

- Acetylation of lysine amino acids in the N-terminal regions of each of the core histones is associated with *activation ofa region of the genome*, and deacetylation leads to *genome silencing*. Histones can also be modified by methylation, Phosphorylation, and Ubiquitination.

- There may be a histone code that specifies how particular combinations of nucleosomes modifications should be interpreted by the genome.

- **Nucleosome repositioning** is required for the *expression of some*, but not all, genes.

- Regions of the genome can be also silenced by **DNA methylation**, the relevant enzymes possible working in *conjunction with histone deacetylase*.

- **Methylation is responsible for genomic imprinting**, which results in *one of a pair of genes* on homologous chromosomes becoming *silenced*, and X inactivation, which leads to the almost *complete inactivation* of one of the X chromosomes in a female nucleus.
