# Chapter 13

# PROTEOMES



## Identifying proteins that interact with one another

*Important data about the function of a proteome can be gained by identifying pairs and groups of proteins that interact with one another.*

### Identifying pairs of interacting proteins

There are several methods for studying protein-protein interactions. Two most important are:

- **Phage display**:
For this a special type of cloning vector is used. It is based on lambda bacteriophage. The vector is designed so that when a new gene is inserted, it is expressed in such a way that its protein product becomes fused with one of the phage coat proteins.

Why the hell would you fuse it with a coat protein anyway? what's the deal?

Doing so makes the inserted gene able to interact with other proteins that the phage encounters.

- **Yeast two-hybrid system**:
Transcription factors are responsible for the expression of genes in eukaryotes (this method is based on eukaryote). And this method makes use of yeast cells that LACK a transcription factor for a reporter gene. The gene is therefore switched off. An artificial gene that codes for the DNA-binding domain of the transcription factor is ligated to the gene for the protein whose interactions we wish to study. After introduction into the yeast, this construct specifies synthesis for a **fusion protein** made up of the DNA-binding domain of the transcription factor attached to the human protein.

[A video showing the two-hybrid analysis](https://www.youtube.com/watch?v=QpZ5Cv7YVvk)

- **Protein array** can also be used, but the two forementioned techniques are the first choice.


### Protein interaction maps display the interactions within a proteome

*The information from phage display, two-hybrid analyses, and other methods for identifying pairs and groups of proteins that associate with one another in a cell enables a **protein interaction map** to the constructed.*

A protein in such a map is displayed as a dot or a node, and the lines or edges between two dots describe interaction between the proteins.

[A map showing an **interactome**](https://upload.wikimedia.org/wikipedia/commons/1/1d/Schziophrenia_PPI.jpg)

#### What interesting features have emerged from these protein interaction maps?

The most intriguing discovery is that each network is built up around a **small number of proteins that have many interactions** and form *hubs* in the network along with a much larger number of proteins with few individual connections.

Interaction maps are all incomplete, never fully comprehending a full proteome. It is unsure if it will ever be able to map out a whole proteome. Despite the limitations of protein-protein mapping techniques, protein interaction maps are proving valuable as a means of probing the link between the proteome and the biochemistry of the cell. In particular, different genes that, when defective, give rise to the same disease often specify proteins that occupy a distinct **disease module** within a network.

*The interaction maps helped discover that same proteins associated with different diseases overlap. Interaction maps also provided important insights into comorbidity, which is the tendency for patients suffering from one disease to display symptoms of another disease.*



## Synthesis and degradation of the components of the proteome

Ribosomes are though to play two active roles:

- Ribosomes *coordinate* protein synthesis by placing the mRNA, tRNAs and associated protein factors in their correct positions relative to one another.

- Components of ribosomes, including the rRNAs, *catalyze* at least some of the chemical reactions that occur during protein synthesis, including the central reaction that results in synthesis of the peptide bond that links two amino acids together.

X-ray crystallography has been responsible for the most exciting insights into ribosome structure.


### The translation of individual mRNAs can also be regulated

In bacteria, several of these **transcription regulation** processes involve attachment of an RNA-binding protein to the target mRNA, the attached protein blocking access to the ribosome binding site and hence *inhibiting formation of the translation initiation complex*. The most frequent sited example involves the operons for the ribosomal protein genes of E. coli.

**RNA-binding protein regulation**

rRNA can switch off further synthesis of mRNA for a ribosomal protein.

Regulation of ferritin protein synthesis in mammals. The iron-response protein binds to the 5'-untranslated region of the ferritin mRNA when iron is NOT PRESENT, preventing the synthesis of ferritin.

Initiation of translation of some bacterial mRNAs can also be regulated by short RNAs that attach to the recognition sequences within the mRNAs, but this does not always result in translation being prevented, as some short RNAs can also activate translation of one or more of their target mRNAs.

### Degradation of the components of the proteome

Why degrade proteins? "*The ability of a proteome to change over time requires the removal of proteins whose functions are no longer required. This removal must be selective and rapid to account for abrupt changes.*"

Ubiquitin attachment targets proteins for degradation. That is the first step. The second step is the ubiquitin-dependent degradation pathway - the proteasome. It breaks down ubiquitinated proteins. The eukaryotic proteasome is a cylinder like multiprotein complex with two caps at the ends. The steps involved in protein degradation requires energy. It is obtained from the hydrolysis of ATP.

## Influence of protein processing on the composition of the proteome

Translation is not the end of the genome expression pathway. The polypeptide that emerges from the ribosome is inactive, and before taking on its functional role in the cell it must undergo post-translational processing. The most fundamental of these is **protein folding** (tertiary structure). The proteome therefore contains not only functional proteins but also a large variety of preprocessed forms, some of which can be retained in their inactive state for substantial periods before processing is completed and the functional protein is produced.

### PROTEIN PROCESSING

## The amino acid sequence contains instructions for the protein folding

All the information a polypeptide needs in order to fold into its 3D-structure is contained within the amino acid sequence.

"Urea disrupts hydrogen bonding." Adding urea disrupts the 3d-structure of some proteins, and taking the urea away lets the proteins fold back again. The same happened with disulfide bridge disruption.

**The molten globule model**: the initial step is the rapid collapse of the polypeptide because of hydrophobic side chains. this collapse is thought to maybe automatically fold into helikses or beta-sheets. In thermodynamic terms, the polypeptide is looked on as passing down a funnel, gradually taking up less random conformations. The number of possible conformations decrease as the polypeptide folds more.

The folding needs help from so called **molecular chaperones**. These are proteins that help other proteins fold. They can be divided into various groups, but two are most important:

- The **Hsp70 chaperones** include the *E. coli* Hslp70 protein cded by the *dnaK* gene and sometimes called DnaK protein.

- The **chaperonins**, the main versions of which are the GroEL/GroES complex present in bacteria and eukaryotic organelles, and TRiC and related structures which are found in eukaryotic cytoplasm and in archaea.

Chaperones does not specify the tertiary structure, but merely help the protein find the correct structure. For the GroEL/GroES chaperones - the polypeptides enter this bullet like cavity shaped protein complex and exits a folded protein.

It seems though that the Hsp70 is the main contributor to help for protein folding. TRiC chaperonins seems to only help 10% of the time.

### Some proteins are activated by proteolytic cleavage

Processing by proteolytic cleavage is common in eukaryotes but less frequent in bacteria. Melittin, a protein in bee venom lyses cells in bees and animals so it must not be active when it is first synthesized by the bee. The precursor *promelittin* has 22 amino acids attached at its N-terminus. The presequence is removed by an exracellular protease that cuts at 11 positions, releasing the active venom protein. The protease does not cleave the active sequence for the venom, since the cleavage follows certain motifs - X-Y where X is alanine, aspartic acid or glutamic acid and Y is alanine or proline. These motifs do nt occur within the active sequence.

A similar process happens with insulin. An active sequence attached to a protein is then cleaved and the active sequence is then activated. it goes from **PREPROINSULIN**, cuts signal peptide at N-terminal --> now **PROINSULIN** folds and creates disulfide bridges --> the chain not taking part is cut at both ends near active sequence --> **INSULIN**.

### Important changes in protein activity can be brought about by chemical modifications

The simplest types of chemical modification involve addition of a small chemical group (for example acetyl, methyl or phosphate group) to an amino acid side chain or to the amino or carboxyl groups of the terminal amino acids in polypeptide.

A more complex modification is called **glycosylation** which involves the attachment of large carbohydrate side chains, called **glycans**, to polypeptides. There are two general types of **glycosylation**:

- **Oxygen-linked glycosylation** is the attachment of a glycan to the hydroxyl group of a serine or threonine.

- **N-linked glycosylation** involves attachment of a glycan to the amino group of the side chain of asparagine.


## BYOND THE PROTEOME

Beyond the proteome comes **metabolome** and **systems biology**

### The metabolism is the complete set of metabolites present in a cell

**Metabolic flux**: the rate of flow of metabolites through the network of pathways that make up the cellular biochemistry. Changes in the metabolome can then be defined in terms of perturbation in the flux of metabolites through one or more parts of the network, providing a very sophisticated description of the biochemical basis for changes in the physiological state. This leads to the possibility of **metabolic engineering**. Biochemical profiling could indicate any unwanted side effects of drug treatment, enabling modifications to be made to the chemical structure of the drug or to its mode of use, so that these side effects are minimized.

### Systems biology provides an integrated description of cellular activity

The emphasis is now not only placed in RNAs, proteins and metabolites which is directed by the genome, but also on the biological systems that result from the coordinated activity of those molecules.

"*Simply transcribing and translating the genes in a operon will not lead to the synthesis of tryptophan. Successful operation of the system required that the biosynthetic enzymes be present at the appropriate place in the cell, in the appropriate relative amounts, at the appropriate time. The system is therefore dependent on factors such as the rate of synthesis of the proteins coded by the genes, correct folding of these proteins into functional enzymes, the rate of degradation of the enzyme molecules, their localization in the cell, and the presence of the necessary amounts of the metabolites that act as substrates and cofactors for tryptophan synthesis. This simple biological system is starting to assume quite considerable complexity.*"

**Operon**: "In genetics, an operon is a functioning unit of DNA containing a cluster of genes under the control of a single promoter."

**Autophagy**: "is the natural, regulated, destructive mechanism of the cell that disassembles unnecessary or dysfunctional components."
